onnx/version_converter/convert.h

```diff
@@ -161,6 +162,25 @@ class DefaultVersionConverter : public BaseVersionConverter {
       registerAdapter(make_unique<CompatibleAdapter>("MaxPool",
         OpSetID(7), OpSetID(8)));
       registerAdapter(make_unique<MaxPool_8_7>());
+      registerAdapter(make_unique<RemoveConsumedInputs>("LeakyRelu",
+        OpSetID(5), OpSetID(6)));
+      registerAdapter(make_unique<CompatibleAdapter>("LeakyRelu",
+        OpSetID(6), OpSetID(5)));
+      registerAdapter(make_unique<BroadcastBackwardCompatibility>("Sub",
+        OpSetID(7), OpSetID(6)));
+      registerAdapter(make_unique<BroadcastForwardCompatibility>("Sub",
+        OpSetID(6), OpSetID(7)));
+      registerAdapter(make_unique<TypeRestriction>("Sub",
+        OpSetID(6), OpSetID(5), broadcast_unallowed_types));
+      registerAdapter(make_unique<RemoveConsumedInputs>("Sub",
+        OpSetID(5), OpSetID(6)));
+      registerAdapter(make_unique<CompatibleAdapter>("PRelu",
+        OpSetID(6), OpSetID(5)));
+      registerAdapter(make_unique<RemoveConsumedInputs>("PRelu",
+        OpSetID(5), OpSetID(6)));
+      registerAdapter(make_unique<CompatibleAdapter>("PRelu",
+        OpSetID(6), OpSetID(7)));
```

Comment:
We need to add a unsqueeze to make sure the second input is in the shape of [C, 1, 1, ...]